app/router.js

```diff
@@ -27,7 +27,7 @@ const Router = EmberRouter.extend({
       const title = this.getWithDefault('currentRouteName', 'unknown');
 
       // this is constant for this app and is only used to identify page views in the GA dashboard
-      const hostname = 'guides.emberjs.com';
+      const hostname = 'octane-guides-preview.emberjs.com';
```

Comment:
Is there a reason we hardcode the hostname at all, vs. getting it from `window.location.hostname`?

Reply:
this is **purely** for google analytics. And hostname isn't what you think it is here 🙈 it's just a flag to allow us to group things in the GA admin